aspnetcore/blazor/forms-validation.md

```diff
@@ -184,6 +243,33 @@ Create a component with the following markup, and use the component just as `Inp
         this, __value => CurrentValueAsString = __value, CurrentValueAsString)" />
 ```
 
+## Work with radio buttons
+
+Blazor doesn't provide a built-in scenario for handling radio button `<input>` elements. Radio button support is under consideration for a future Blazor release. The following workaround is recommended:
```

Comment:
```suggestion
When working with radio buttons in a form, data binding is handled differently because radio buttons are evaluated as a group. The value of each radio button is fixed, but the value of the radio button group is the value of the selected radio button. The example below shows how to handle data binding for a radio button group:
```

Reply:
We should probably also add a link to this section from the data binding section of the Components article.